cms/envs/common.py

```diff
@@ -563,6 +563,7 @@
 AWS_SES_REGION_ENDPOINT = 'email.us-east-1.amazonaws.com'
 AWS_ACCESS_KEY_ID = None
 AWS_SECRET_ACCESS_KEY = None
+AWS_SECURITY_TOKEN = None
```

Comment:
Is there any expiration for this?

Reply:
Yes, this will be generated every time the assume role is run.

Reply:
how often will we have to run assume role? Like previously we had to run it after every hour because it got expired

Reply:
Same case here. The credentials are valid for 60 minutes only.